ext/php7/serializer.c

```diff
@@ -16,9 +18,16 @@
 #include "mpack/mpack.h"
 #include "span.h"
 
+// we should use zval_to_uint64 from random.h
+#include "random.h"
+
 ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
 
-static int msgpack_write_zval(mpack_writer_t *writer, zval *trace TSRMLS_DC);
+static const char *trace_id_keyword = "trace_id";
```

Comment:
Either use the variable everywhere (i.e. also in the add_assoc_string below) or not at all.

Reply:
Very good point, did not catch that!